.travis/check-doc-change

```diff
@@ -11,6 +11,16 @@ case ${TRAVIS_EVENT_TYPE} in
 push)
     if [[ "${TRAVIS_BRANCH}" = master ]]
     then
+        # You can set master skipping strategy by using environment variable
+        # ${MASTER_SKIP_STRATEGY} : auto (default), skip, noskip
+        # auto will find appropriate merge base
+        if [[ "${TRAVIS_MASTER_SKIP_STRATEGY:-auto}" -ne "auto" ]]
```

Comment:
What's TRAVIS_MASTER_SKIP_STRATEGY?
Is it a Travis provided variable or the custom variable?
And when do you want to set it `skip` or `noskip`?

Reply:
It doesn't seem useful for now. I'll remove it.